web-app/src/auth/auth.controller.ts

```diff
@@ -0,0 +1,27 @@
+import {
+  Controller,
+  Post,
+  Request,
+  UseGuards,
+  Get
+} from '@nestjs/common';
+import { AuthGuard } from '@nestjs/passport';
+import { AuthService } from './auth.service';
+
+@Controller('api/auth')
+export class AuthController {
+
+  constructor(private authService: AuthService) { }
+
+  @UseGuards(AuthGuard('local'))
+  @Post('/login')
+  async login(@Request() req) {
```

Comment:
Can't you be more specific about what is expected in user?

Reply:
I don't know how to do it, because the user object is injected from the guard mechanism and don't know how to retrieve the returned type...

Reply:
I propose to leave it as is for the moment!